src/services/gpt.js

```diff
@@ -60,6 +62,7 @@ export function refreshSlot({
 * @param {function} fn - Accepts a function to push into the Prebid command queue.
 **/
 export function queueGoogletagCommand(fn) {
+  sendLog(`The function about to be pushed to window.googletag.cmd is ${fn}`);
```

Comment:
I haven't ran it locally yet, can you provide an example of what this line outputs?